nym-api/src/support/http/helpers.rs

```diff
@@ -1,11 +1,72 @@
 // Copyright 2024 - Nym Technologies SA <contact@nymtech.net>
 // SPDX-License-Identifier: GPL-3.0-only
 
+use axum::extract::{ConnectInfo, Request};
+use axum::middleware::Next;
+use axum::response::IntoResponse;
+use colored::Colorize;
+use http::header::{HOST, USER_AGENT};
+use http::HeaderValue;
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
+use std::net::SocketAddr;
+use std::time::Instant;
+use tracing::info;
 
 #[derive(Serialize, Deserialize, FromForm, Debug, JsonSchema)]
 pub struct PaginationRequest {
     pub page: Option<u32>,
     pub per_page: Option<u32>,
 }
+
+// TODO dz move this fn to a "common" place so this can be exported ?
+pub async fn logger(
```

Comment:
I swear I already put this code alongside other common axum helpers in http-common crate on my initial axum-migration branch 🤔

Reply:
yep, moved